tillerinobot/src/main/java/tillerino/tillerinobot/lang/Slovak.java

```diff
@@ -0,0 +1,243 @@
+		return new Message("Poď sem, ty!")
+			.then(new Action("objíma " + apiUser.getUserName()));
+	}
+
+	@Override
+	public String help() {
+		return "Ahoj! Ja som robot, ktorý zabil Tillerina a prevzal jeho účet. Robím si srandu, ale veľmi často používam jeho účet."
+				+ " [https://twitter.com/Tillerinobot status a aktualizácie]"
+				+ " - [https://github.com/Tillerino/Tillerinobot/wiki príkazy]"
+				+ " - [http://ppaddict.tillerino.org/ ppaddict]"
+				+ " - [https://github.com/Tillerino/Tillerinobot/wiki/Contact kontakt]";
+	}
+
+	@Override
+	public String faq() {
+		return "[https://github.com/Tillerino/Tillerinobot/wiki/FAQ Často kladené otázky]";
+	}
+	
+	@Override
+	public String featureRankRestricted(String feature, int minRank, OsuApiUser user) {
+		return "Je mi ľúto, ale v tejto chvíli " + feature + " je sprístupnená len pre hráčov, ktorí prekonali rank " + minRank + ".";
+	}
+	
+	@Override
+	public String mixedNomodAndMods() {
+		return "Čo myslíš tým, že si zmiešal bez módu s módmi?";
+	}
+	
+	@Override
+	public String outOfRecommendations() {
+		return "[https://github.com/Tillerino/Tillerinobot/wiki/FAQ#the-bot-says-its-out-of-recommendations-what-do"
+				+ " Doporučil som ti všetko, čo mi prišlo na um]."
+				+ " Skús ostatné možnosti doporučenia, alebo použi !reset. Pokiaľ si si neistý(á), pozri sa na !help.";
+	}
+
+	@Override
+	public String notRanked() {
+		return "Vyzerá to tak, že táto mapa nie je hodnotená.";
+	}
+
+	@Override
+	public String invalidAccuracy(String acc) {
+		return "Neplatná presnosť: \"" + acc + "\"";
+	}
+
+	@Override
+	public Response optionalCommentOnLanguage(OsuApiUser apiUser) {
+    /*
+		 * TRANSLATION NOTE: This line is sent to the user right after they have
+		 * chosen this Language implementation. The English version refers to
+		 * itself as the default version ("just the way I am"), so translating
+		 * the English message doesn't make any sense.
+		 * 
+		 * Instead, we've been using the line
+		 * "*Translator* helped me learn *Language*." in translations. Replace
+		 * *Translator* with your osu name and *Language* with the name of the
+		 * language that you are translating to, and translate the line into the
+		 * new language. This serves two purposes: It shows that the language
+		 * was changed and gives credit to the translator.
+		 * 
+		 * You don't need to use the line above, and you don't have have to give
+		 * yourself credit, but you should show that the language has changed.
+		 * For example, in the German translation, I just used the line
+		 * "Nichts leichter als das!", which translates literally to
+		 * "Nothing easier than that!", which refers to German being my first
+		 * language.
+		 * 
+		 * Tillerino
+		 * 
+		 * P.S. you can put a link to your profile into the line like this:
+		 * [https://osu.ppy.sh/u/2070907 Tillerino]
+		 */
+		return new Message("[https://osu.ppy.sh/u/BramboraSK BramboraSK] mi pomohol naučiť sa slovensky.");
```

Comment:
The numeric user id will work for longer :)

```suggestion
		return new Message("[https://osu.ppy.sh/u/8543258 BramboraSK] mi pomohol naučiť sa slovensky.");
```